components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/impl/WebAppAuthenticatorImpl.java

```diff
@@ -119,13 +130,13 @@ private boolean validateScopes(Message message, AccessTokenInfo tokenInfo) {
                             }
                             return true;
                         }
-                    }
-                    else {
+                    } else {
                         if (log.isDebugEnabled()) {
-                        log.debug("Scope not defined in swagger for matching resource. So consider as anonymous permission and" +
-                                "let request to");
-                            return true;
+                            log.debug(
```

Comment:
Log message doesn't mention the resource path that is being validated.
Log message isn't complete. "... let request to." Let request to do what?